internal/test/smoke/rbac_permission_test.go

```diff
@@ -239,13 +239,14 @@ func (s *SuiteRbacPermission) TestReadPermission() {
 			Name:     "Test idmsvc:domain:create",
 			Given:    s.prepareDomainIpaCreate,
 			Then:     s.doTestDomainIpaCreate,
-			Expected: http.StatusUnauthorized,
+			Expected: http.StatusCreated,
 		},
 		{
-			Name:     "Test Update Agent idmsvc:domain:update",
-			Given:    s.prepareDomainIpa,
-			Then:     s.doTestDomainIpaUpdate,
-			Expected: http.StatusUnauthorized,
+			Name:  "Test Update Agent idmsvc:domain:update",
+			Given: s.prepareDomainIpa,
+			Then:  s.doTestDomainIpaUpdate,
+			// System requests are identified by its certificate
+			Expected: http.StatusOK,
```

Comment:
Same here. Also I don't understand much the comment `// System requests are identified by its certificate` - how it is related to this test.

Reply:
thanks! I have restructured the table, but that set of tests make sense in a different suite dedicated to the System endpoints. I will refactor! :+1: